### DISCUSSION TITLE
Replies to Discussions via email don't end up in threads

### DISCUSSION THREAD

**Discussion by davidkohn88** [community/community#3287] [Discussions]
When people reply to a comment, either within a thread or not, the reply goes to the main discussion as a separate item rather than appearing in the thread. It would be nice if it appeared in the thread, or if there were an option to move a comment to a thread so that we can keep replies more organized.

**Comment by PranamBhat** [comment]
Hello @davidkohn88 

As we can see, this feature already exists and is working as expected.

![image](https://user-images.githubusercontent.com/32120033/108156479-bed05980-7106-11eb-9e1f-31da54c3d1ae.png)

**Reply by davidkohn88** [reply]
Did you reply via the email? Because it's definitely not working in some of the discussions I have...

**Reply by davidkohn88** [reply]
If you reply in the thread, it totally works, but if you reply via an email notification it goes to the main thread.

**Reply by JLockerman** [reply]
@davidkohn88 do you have and example you can share?

**Reply by miharekar** [reply]
@JLockerman right here: https://github.com/miharekar/decent-visualizer/discussions/14#discussioncomment-460861

**Comment by davidkohn88** [comment]
Does this work?

**Reply by davidkohn88** [reply]
So the above was a reply via email to @JLockerman, https://github.com/github/feedback/discussions/3287#discussioncomment-378663 in the thread above, you'll note that it appeared as a separate heading rather than in the thread.

**Comment by davidkohn88** [comment]
If I reply by email like this again to https://github.com/github/feedback/discussions/3287#discussioncomment-378663, I think I will end up with a new post
rather than ending up in a thread for a given question.
I think in general replies should default to being in thread, with
potentially an option to open to a new post or whatever the higher level
item is called.

**Reply by davidkohn88** [reply]
Yep. this was also a new post rather than being in the thread.

**Comment by PranamBhat** [comment]
True. It appears in a separate thread! This should not be like this!!!

**Reply by davidkohn88** [reply]
Can you take down your original reply, I think others see it and think that the issue has been resolved when it has not been!

**Reply by mattfysh** [reply]
Any updates to this issue? Very strange behaviour

**Comment by introt** [comment]
Bump, still broken: https://github.com/zim-desktop-wiki/zim-desktop-wiki/discussions/1613#discussioncomment-1374528

**Reply by introt** [reply]
Ping @JLockerman

**Comment by Tryme69** [comment]
https://github.com/bugsnag/bugsnag-js/blob/tms/rn-vendoring/.github/support.md

**Reply by mattfysh** [reply]
Page not found

**Comment by biajoeknee** [comment]
still happening for me, should i be formatting an email reply specially so that it appears in the original thread?

**Comment by vkbo** [comment]
The threading feature is so hopeless that I wish there was an option to disable it and just have linear discussions like on issues. Half the people using the discussions on my project don't realise they are threaded at all and will respond all over the place in the first box they notice. Then there are the email responses ending up in the wrong place.

This is a poorly designed UI and it is incredibly frustrating to admin it because you have no moderator options to move responses to the correct place or anything like that. Discussions have been a broken half-assed beta feature for years, and I mostly just want to turn it off again.

**Comment by Betty856** [comment]
When people reply to a comment, either within a thread or not, the reply goes to the main discussion as a separate item rather than appearing in the thread. It would be nice if it appeared in the thread, or if there were an option to move a comment to a thread so that we can keep replies more organized. For those interested in photo editing,[remini for pc crack](https://reminiapk.io/remini-mod-apk-pc/) offers advanced features without limitations, providing a more organized and efficient editing process.

**Comment by Frederick888** [comment]
Personally I don't use Discussions that often but conversations in PRs have the same issue, which annoys me way much more.

Is there another report for PRs? If not, @davidkohn88 could you please rename the title here to 'Replies to Discussions/PRs...'? It's probably the same underlying issue so it's better to consolidate them.
